flink-runtime/src/main/java/org/apache/flink/runtime/resourcemanager/ResourceManager.java

```diff
@@ -582,16 +582,41 @@ public void notifySlotAvailable(
 	}
 
 	@Override
-	public CompletableFuture<TransientBlobKey> requestTaskManagerFileUpload(ResourceID taskManagerId, FileType fileType, Time timeout) {
-		log.debug("Request file {} upload from TaskExecutor {}.", fileType, taskManagerId);
+	public CompletableFuture<TransientBlobKey> requestTaskManagerFileUploadByType(ResourceID taskManagerId, FileType fileType, Time timeout) {
+		log.debug("Request file which type is {}  upload from TaskExecutor {}.", fileType, taskManagerId);
 
 		final WorkerRegistration<WorkerType> taskExecutor = taskExecutors.get(taskManagerId);
 
 		if (taskExecutor == null) {
-			log.debug("Requested file {} upload from unregistered TaskExecutor {}.", fileType, taskManagerId);
+			log.debug("Requested which type is {} upload from unregistered TaskExecutor {}.", fileType, taskManagerId);
```

Comment:
```
log.debug("Request upload of log file {} from TaskExecutor {}.", fileType, taskManagerId);
```

Reply:
As here taskExecutor is null, so we need add unregistered for TaskExecutor.
```code
log.debug("Request upload of file {} from unregistered TaskExecutor {}.", fileType, taskManagerId);
```